recipes/mongoquery/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "1.1.0" %}
+
+package:
+    name: mongoquery
+    version: {{ version }}
+
+source:
+    url: https://github.com/kapouille/mongoquery/archive/v{{ version }}.tar.gz
+    fn: mongoquery-v{{ version }}.tar.gz
+    sha256: d1e87f6038ea074273610b6258e5af7477de61c782f6424fc7ecf3416df3e645
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+    build:
+        - setuptools
+        - python
+    run:
+        - python
+
+test:
+    imports:
+        - mongoquery
+    requires:
+        - pytest
+    commands:
+        - py.test {{ environ.SRC_DIR }}/tests -vrsx
+
+about:
+    home: https://github.com/kapouille/mongoquery
+    license: The Unlicense
```

Comment:
@ocefpaf @jakirkham Is this the way you would like to refer this license?

[Here] is the license that accompanies the git repo